package/src/components/MessageOverlay/MessageOverlay.tsx

```diff
@@ -441,6 +443,14 @@ const MessageOverlayWithContext = <
                             />
                           )
                         );
+                      case 'poll': {
+                        const pollId = message.poll_id;
+                        return Poll && pollId && ownCapabilities ? (
+                          <OwnCapabilitiesProvider key={`poll_${pollId}`} value={ownCapabilities}>
```

Comment:
It's [because of this](https://github.com/GetStream/stream-chat-react-native/blob/e81c6bc9a0e5734e7af02d59f8d644e7dbde36cb/package/src/components/MessageOverlay/MessageOverlay.tsx#L385).